lib/std/start.zig

```diff
@@ -330,17 +330,36 @@ fn _start() callconv(.Naked) noreturn {
             ,
             .mips, .mipsel =>
             \\ move $fp, $0
+            \\ bal 1f
+            \\ .gpword .
+            \\ .gpword %[posixCallMainAndExit]
+            \\ 1:
+            \\ lw $gp, 0($ra)
+            \\ subu $gp, $ra, $gp
+            \\ lw $25, 4($ra)
+            \\ addu $25, $25, $gp
             \\ move $ra, $0
             \\ move $a0, $sp
             \\ and $sp, -8
-            \\ j %[posixCallMainAndExit]
+            \\ subu $sp, $sp, 16
+            \\ jalr $25
             ,
             .mips64, .mips64el =>
             \\ move $fp, $0
+            \\ .balign 8
```

Comment:
This is needed because early MIPS versions don't support misaligned loads. Without this directive, the hidden `nop` that's inserted after the `bal` (to fill the delay slot) would cause the two doublewords below to be misaligned.

This problem doesn't exist on mips32 because instructions and words have the same width.